docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -244,6 +244,7 @@ This document provides Cobalt users instructions for initializing and integratin
 
     * Open the project from your favorite IDE and navigate to infrastructure templates `./infra/templates` directory.
     * Manually delete template directories not needed for your enterprise.
+    * The CI/CD pipeline needs to detect a code change to run tests. Add a comment or extra line to a TF or Go file in order to force tests to run.
```

Comment:
note: @awkwardindustries is working on making this configurable at the pipeline level to avoid this. Can you two sync in order to get the right documentation?

Reply:
@nmiodice do you think the work that @awkwardindustries is doing will negate this PR?

Reply:
@iphilpot @nmiodice : PR #280 supports forcing the pipeline execution through by setting a `FORCE_RUN` environment variable to `true` (in the correct environment-specific Environment Variables variable group). I'd recommend updating this note to explain those options more clearly -- set an environment variable, or add a comment or extra line to a TF or Go file, etc. I don't think it negates the clarity provided by the rest of the adjustments in this PR though. :)

Reply:
We're going with the `FORCE_RUN` environment variable, so I'll make a suggestion to clear up this note shortly.

Reply:
```suggestion
    * The CI/CD pipeline needs to detect a code change to run the template-specific build and release jobs (in their respective stages). To force the template build and release to run, you may add a `FORCE_RUN` environment variable with a value of `true` to your *devint Environment Variables* variable group. You may also add a comment or extra line to a TF or Go file within the template in order for the pipeline script to detect a change without adding any additional override flags.
```

Reply:
I refer to FORCE_RUN on line 239 and line 255 of both the ADD_PAT_OWNER and ADD_PAT_OWNER_CLI respectively.